site/content/en/latest/tasks/extensibility/envoy-patch-policy.md

```diff
@@ -82,6 +82,35 @@ data:
 
 {{< boilerplate rollout-envoy-gateway >}}
 
+## XDS Name Scheme V2
+
+Starting from v1.5, Envoy Gateway uses version 2 of the xDS name scheme when generating xDS resources.
+Because [EnvoyPatchPolicy][] relies on specific xDS resource names, it’s important to use the correct naming format when authoring a patch policy.
+
+The name format in the old version of scheme:
+* Listener name: `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`, example: `default/eg/http`
+* RouteConfig/FilterChain name: `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`, example: `default/eg/http`
+* FilterChain name: `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`, example: `default/eg/http`
+* VirtualHost name: `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>/<VirtualHost>`, example: `default/ef/http/www_example_com`
+* HCM StatPrefix: `<ApplicationProtocol>/<ContainerPort>`, example: `http-10080` or `https-10443`
+
+Name format in version 2 of the scheme:
+* Listener name: `<Protocol>-<Port>`, example: `tcp-80`
+* RouteConfig name:
+   * HTTP Listener RouteConfig name: `<Port>`, example: `80`
+   * HTTPS Listener RouteConfig name: `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`, example: `default/eg/http` (same as the old version)
+* FilterChain name:
```

Comment:
FilterChain name, VirtualHost name and HCM StatPrefix are probably not needed for EnvoyPatchPolicy, but it would be good to have one place to show all the changes.